tests/x11/sshxterm.pm

```diff
@@ -15,16 +15,20 @@ sub run() {
     my $self = shift;
     mouse_hide(1);
     x11_start_program("xterm");
-    script_run("ssh -XC root\@localhost xterm");
-    type_string "yes\n";
-    wait_idle 6;
+
+    # check last incoming ssh traffic, wait until the connection is opened from localhost.
+    # "ESTAB", AND ::1:ssh, AND dport(destination-port) for control with exactitude, that a inc. port from ssh only from localhost is open.
+    # at this moment type password with serenity.
+    script_run("for i in {1..100}; do /usr/sbin/ss -at dport = :ssh | tail -n1 >/dev/$serialdev; sleep 2; done & ");
+    script_run("ssh  -o  \"StrictHostKeyChecking no\" -XC root\@localhost xterm ");
+    wait_serial('ESTAB.*::1:ssh');
     type_string "$password\n";
     sleep 2;
     for (1 .. 13) { send_key "ret" }
     $self->set_standard_prompt();
     type_string "echo If you can see this text, ssh-X-forwarding  is working.\n";
     sleep 2;
-    assert_screen 'test-sshxterm-1', 3;
+    assert_screen 'test-sshxterm-1', 5;
```

Comment:
better change to use default timeout, see explanation here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/945